components/eam/src/physics/cam/physics_types.F90

```diff
@@ -574,6 +575,8 @@ subroutine physics_state_check(state, name)
          varname="state%t",         msg=msg)
     call shr_assert_in_domain(state%pv(:ncol,:),        is_nan=.false., &
          varname="state%pv",         msg=msg)
+    call shr_assert_in_domain(state%pt(:ncol,:),        is_nan=.false., &
+         varname="state%pv",         msg=msg)
```

Comment:
This looks like it should use `varname=“state%pt”` instead of `varname=“state%pv”`.

Reply:
agreed; fixed